### DISCUSSION TITLE
Pricing Page for Flex

### DISCUSSION THREAD

**Discussion by sharfatehi** [twilio-labs/paste#1890] [General] [answered]
1. For disclosures are we able to change the background color when they are in the collapsed state?
I had changed the background color to grey in my mocks in Figma, but my developer said she was unable to change the color with the pattern.

2. I have a pricing example using rounded ends, but my developer said she wasn't able to round the corners past a certain point. Is this something we can do? We are certain this is something our customers will want.

Examples attached: 
First image is what the disclosures and example should look like, last two are what has been developed.

<img width="495" alt="Screen Shot 2021-09-27 at 10 40 42 AM" src="https://user-images.githubusercontent.com/78043210/134952347-bc61138e-4c48-48af-9f9c-5424c6b85f8b.png">

**Comment by sharfatehi** [comment]
<img width="668" alt="Screen Shot 2021-09-27 at 10 43 36 AM" src="https://user-images.githubusercontent.com/78043210/134952652-376c5772-c219-4e7f-9d78-41bcecf6576e.png">

**Comment by sharfatehi** [comment]
<img width="973" alt="Screen Shot 2021-09-27 at 10 44 07 AM" src="https://user-images.githubusercontent.com/78043210/134952671-064c203d-cf4f-4a93-a960-c809263c1957.png">

**Comment by loreina** [comment, marked as answer]
Hi @sharfatehi,

1. The background color for Disclosure isn't currently modifiable. We're looking to do another design pass of Disclosure and it would be helpful to understand the use cases for it here. Can you share more about why this would require a background color? Would you be able to share your Figma files as well? 

2. [These](https://paste.twilio.design/tokens/#radii) are our available radii tokens which means we unfortunately can't support the fully rounded corners of your design yet. We're looking to add more radii tokens though and would love input from your team on what customers have said they wanted.

Happy to chat about this at [office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) too. Feel free to tag your engineer in this discussion or have them chime in as well.

**Comment by sharfatehi** [comment]
Thanks for the updates!

Use case for the disclosure bg color, is for visibility. We have them under a bit of text and I wanted them to be more distinguishable to the user.

https://www.figma.com/file/FsOLjTnA6PuavfXdUonCQv/Pricing?node-id=0%3A1

I can follow up about the radii token.

**Comment by andioneto** [comment]
@sharfatehi Hi there 👋 I wanted to follow up and see if you need any more assistance or have any more unanswered questions? If not we can mark this as answered. Thanks!

**Comment by sharfatehi** [comment]
I have everything I need, thank you!
